src/components/list-item-actions/list-item-actions.tsx

```diff
@@ -0,0 +1,29 @@
+import { List } from '@patternfly/react-core';
+import React from 'react';
+import { StatefulDropdown } from '../patternfly-wrappers/stateful-dropdown';
+
+interface IProps {
+  kebabItems?: React.ReactNode[];
+  buttons?: React.ReactNode[];
+}
+export class ListItemActions extends React.Component<IProps> {
+  render() {
+    return (
+      <td
+        style={{
+          paddingRight: '0px',
+          textAlign: 'right',
+          display: 'flex',
+          justifyContent: 'flex-end',
+        }}
+      >
+        {this.props.buttons?.length && <List>{this.props.buttons} </List>}
+        {this.props.kebabItems?.length && (
+          <div data-cy='kebab-toggle'>
+            <StatefulDropdown items={this.props.kebabItems.filter(Boolean)} />{' '}
```

Comment:
Oh and one more thing, to support logic like `<ListItemActions kebabItems={[ maybe && Button ])` without possibly rendering `false` in the dropdown when `maybe === false`, we should filter the buttons and kebabItems first...

```suggestion

export class ListItemActions extends React.Component<IProps> {
  render() {
    const buttons = this.props.buttons?.filter(Boolean);
    const kebabItems = this.props.kebabItems?.filter(Boolean);

    return (
      <td
        style={{
          paddingRight: '0px',
          textAlign: 'right',
          display: 'flex',
          justifyContent: 'flex-end',
        }}
      >
        {buttons?.length && <List>{buttons} </List>}
        {kebabItems?.length && (
          <div data-cy='kebab-toggle'>
            <StatefulDropdown items={kebabItems} />{' '}
```